items/disqus-lazy-load/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "Disqus Lazy Load",
+	"version": "",
+	"author_username": "ikutaa46",
+	"release_date": "2018",
+	"license": "free",
+	"price_usd": "0",
+	"download_url": "https://rawcdn.githack.com/ikutaa46/ikutaa46.github.io/master/bludit/plugin/disqus-lazy-load.zip",
```

Comment:
The plugin is for Bludit v3.x ? if that the field for download is `download_url_v3`

Reply:
i have updated this metadata.json.

1. download_url is for Bludit v2
2 download_url_v3 is for Bludit v3.